content-services/content-sources-frontend
# Fixes 1869: Admin Task Interface

## Summary

Added interface to view tasks across all accounts and organizations at `/insights/content/admin-tasks`

## Testing steps

Navigate to the main content page at `/insights/content` and click on the "Admin tasks" tab. The tab will only appear if the backend config has the admin_tasks feature enabled:

```
features:
  admin_tasks:
    enabled: true
    accounts: ["your-account-id"]
```
